src/kemal/middleware/filters.cr

```diff
@@ -28,6 +33,13 @@ module Kemal::Middleware
     private def radix_path(type : Symbol, path)
       "/#{type}#{path}"
     end
+
+    class BeforeFilterAlreadyDefinedException < Exception
+    end
```

Comment:
How would I interpolate the `path` then ? (I'm sorry I'm still very new to this)